standalone/examples.go

```diff
@@ -0,0 +1,16 @@
+package standalone
+
+// Example model of an example gRPC request
+type Example struct {
+	Name    string `json:"name"`
+	Request struct {
+		Timeout  string `json:"timeout"`
```

Comment:
Instead of making this whole structure stringly-typed, we should provide more typing, to prevent bad data from being supplied.

For this we want a `time.Duration`. But it needs to be wrapped in a type with a custom `MarshalJSON` method to marshal it to a JSON string in the format `%fs` (e.g. floating point duration with a trailing "s" indicating a unit of seconds).

Reply:
Makes sense to me, if we complicate encoding rules, I would add back the decoding helpers as well to make it easy for users to handle example files.

Reply:
I wasn't suggesting we complicate anything -- users should still be able to use `json.Marshal` to read/write files. We put the complexity behind the `JSONMarshaler` interface abstraction. Also, I'm just not a fan of helpers that just call `io.ReadFile` and then call something else. That's so trivial for caller to do that it doesn't file like it carries its weight in a library.

Reply:
Sorry if this sounded defensive, it was not the intent :). For a library the requirements are tighter, it makes sense to have better tools around examples. 

Please have a look at the current draft, I plan to add more tests and handle gRPC payload.